docs/upgrade/UPGRADE-unreleased.md

```diff
@@ -75,6 +75,13 @@ There you can find links to upgrade notes for other versions too.
     - stop using the deprecated method `\Shopsys\FrameworkBundle\Model\Pricing\PriceCalculation::getVatCoefficientByPercent()`, use `PriceCalculation::getVatAmountByPriceWithVat()` for VAT calculation instead
     - if you want to customize the VAT calculation (eg. revert it back to the previous implementation), extend the service `@Shopsys\FrameworkBundle\Model\Pricing\PriceCalculation` and override the method `getVatAmountByPriceWithVat()`
     - if you created new tests regarding the price calculation they might start failing after the upgrade - in such case, please see the new VAT calculation and change the tests expectations accordingly
+- use automatic wiring of Redis clients for easier checking and cleaning ([#1161](https://github.com/shopsys/shopsys/pull/1161))
+    - if you have redefined the service `@Shopsys\FrameworkBundle\Component\Redis\RedisFacade` or `@Shopsys\FrameworkBundle\Command\CheckRedisCommand` in your project, or you instantiate the classes in your code:
+        - instead of instantiating `RedisFacade` with an array of cache clients to be cleaned by `php phing redis-clean`, pass an array of all redis clients and another array of redis clients you don't want to clean (eg. `global` and `session`)
```

Comment:
what about changing the argument name ?
```yaml
Shopsys\FrameworkBundle\Component\Redis\RedisFacade:`
    arguments:
        $cacheClients -> $allClients: 
```
will this work if they override service in ShopBundle ?

Reply:
Configuring it with `$cacheClients:` would fail. I'm not sure how to deal with this because renaming parameters is not covered in our BC promise (and as such we probably shouldn't recommend users to use argument names in their configuration...)

Reply:
yep this is only problem, in other way jobs for no-bc-break-checks passed so :+1: